Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volume-clone.md

```diff
@@ -17,8 +17,16 @@ for more info.
 
 In [pvc-clone](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pvc-clone.yaml),
 `dataSource` should be the name of the `PVC` which is already created by RBD
-CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim` and also storageclass
-should be same as the source `PVC`.
+CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim`.
+The `storageClassName` can be any RBD storageclass (not necessarily same as Parent PVC)
+
+!!!note
+    * `provisinor` should be same for both the Parent PVC and the Clone PVC.
```

Comment:
What if the provisioner is not the same? Is there a scenario for that? If not, let's use "must" instead of "should"
```suggestion
    * `provisioner` must be same for both the Parent PVC and the Clone PVC.
```